lib/HTML/FormBuilder.pm

```diff
@@ -47,22 +47,7 @@ sub BUILDARGS {
     my %args = (@args % 2) ? %{$args[0]} : @args;
 
     # set default class
-    my @class_names = qw( fieldset_group
-        no_stack_field_parent
-        row_padding
-        fieldset_footer
-        comment
-        row
-        extra_tooltip_container
-        backbutton
-        required_asterisk
-        inputtrailing
-        label_column
-        input_column
-        hide_mobile
-    );
-
-    my %classes = map { $_ => $_ } @class_names;
+    my %classes = %{$HTML::FormBuilder::Base::CLASSES};
     %classes = (%classes, %{$args{classes} || {}});
 
     $args{classes} = \%classes;
```

Comment:
can be done with less copying data around:

```perl
if ($args{classes}) {
	$args{classes} = {%{$args{classes}, %$HTML::FormBuilder::Base::CLASSES};
} else {
	$args{classes} = $HTML::FormBuilder::Base::CLASSES;
	# or if you want to avoid accidental modification: (this copies)
	$args{classes} = {%$HTML::FormBuilder::Base::CLASSES};
}
```